packages/compiler-cli/linker/test/linker_import_generator_spec.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {LinkerImportGenerator} from '../src/linker_import_generator';
+
+const ngImport = {};
```

Comment:
If this has some properties it's harder to trick the `toEqual` into accepting another empty object.

```suggestion
const ngImport = {ngImport: true};
```